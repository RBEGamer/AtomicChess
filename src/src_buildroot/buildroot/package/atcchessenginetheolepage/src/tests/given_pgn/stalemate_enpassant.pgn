[Event "YAKAfight"]
[Site "lab"]
[Date "2013.03.04"]
[Round "1"]
[White "lepage_a"]
[Black "chen_a"]
[Result "1/2-1/2"]
[Time "23:16:30"]

1. e2-e4 b7-b5 2. d2-d3 c7-c5 3. c2-c4 b5-b4 4. a2-a4 b4xa3 5. b2xa3 Nb8-c6
6. h2-h4 Bc8-a6 7. Ng1-h3 Qd8-a5+ 8. Nb1-c3 O-O-O 9. Qd1-a4 Qa5xa4 10.
Nc3xa4 Ba6xc4 11. Na4xc5 Bc4-a2 12. a3-a4 a7-a6 13. Nc5xa6 d7-d5 14. a4-a5
d5xe4 15. Na6-c7 Kc8xc7 16. a5-a6 g7-g5 17. a6-a7 g5-g4 18. a7-a8=Q g4xh3
19. Qa8xa2 e4-e3 20. f2xe3 Ng8-f6 21. Bf1-e2 h7-h5 22. O-O Rd8xd3 23.
Be2xd3 Nf6-e4 24. Bd3xe4 Nc6-d4 25. e3xd4 Kc7-b6 26. d4-d5 e7-e5 27. d5xe6
f7-f5 28. e6-e7 f5xe4 29. e7xf8=Q e4-e3 30. Qf8xh8 h3-h2+ 31. Kg1xh2 e3-e2
32. Qa2-c4 e2-e1=Q 33. Rf1xe1 Kb6-b7 34. g2-g3 Kb7-b6 35. Re1-e7 1/2-1/2

