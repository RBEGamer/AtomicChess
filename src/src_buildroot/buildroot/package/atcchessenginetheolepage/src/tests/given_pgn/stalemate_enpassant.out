[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 1 rank: 3
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 1 rank: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[check]	: color: 0
[move]	board::PieceType: 4
[move]	board::PieceType: 0
[queenside castling]	color: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[piece_taken]	 piece: 1 file: 1 rank: 4
[move]	board::PieceType: 4
[piece_taken]	 piece: 1 file: 1 rank: 4
[move]	board::PieceType: 3
[piece_taken]	 piece: 5 file: 3 rank: 4
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 3 rank: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 1 rank: 6
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 5 rank: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 0
[piece_taken]	 piece: 4 file: 3 rank: 7
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_promoted]	 piece: 1 file: 1 rank: 8
[move]	board::PieceType: 5
[piece_taken]	 piece: 4 file: 8 rank: 3
[move]	board::PieceType: 1
[piece_taken]	 piece: 3 file: 1 rank: 2
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 5 rank: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 0
[kingside castling]	color: 0
[move]	board::PieceType: 2
[piece_taken]	 piece: 5 file: 4 rank: 3
[move]	board::PieceType: 3
[piece_taken]	 piece: 2 file: 4 rank: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[piece_taken]	 piece: 4 file: 5 rank: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[piece_taken]	 piece: 4 file: 4 rank: 4
[move]	board::PieceType: 0
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 5 rank: 6
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 3 file: 5 rank: 4
[move]	board::PieceType: 5
[piece_taken]	 piece: 3 file: 6 rank: 8
[piece_promoted]	 piece: 1 file: 6 rank: 8
[move]	board::PieceType: 5
[move]	board::PieceType: 1
[piece_taken]	 piece: 2 file: 8 rank: 8
[move]	board::PieceType: 5
[check]	: color: 0
[move]	board::PieceType: 0
[piece_taken]	 piece: 5 file: 8 rank: 2
[move]	board::PieceType: 5
[move]	board::PieceType: 1
[move]	board::PieceType: 5
[piece_promoted]	 piece: 1 file: 5 rank: 1
[move]	board::PieceType: 2
[piece_taken]	 piece: 1 file: 5 rank: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 5
[move]	board::PieceType: 0
[move]	board::PieceType: 2
[pat]	: color: 1
[draw]	
[game_finished]	
